docs/api/contrib/index.rst

```diff
@@ -1,23 +1,12 @@
 .. -*- mode: rst -*-
 
-Contrib
-==========================
+Yellowbrick Contrib
+===================
 
-The yellowbrick contrib contains a variety of extra tools and experimental visualizers that are outside of core support or are still in development.
-
-
-Classifiers
```

Comment:
@ndanielsen I had to get rid of this because it was messing up the TOC outline on the sidebar. Not sure how to fix it ... I figured once we had more contrib classifier modules we could move them into their own directory.

Reply:
good to know, that can be a TODO